### PR TITLE
docs: add description of how to install jira cloud plugin

### PR DESCRIPTION
**Pull request by daveleek**
## About the changes

Adds more information to the installation section of our Jira Cloud plugin

image

### CODE REVIEW

File: website/docs/reference/api/unleash/unleash-api.info.mdx
Comment (thomasheartman) [inline website/docs/reference/api/unleash/unleash-api.info.mdx:1]: Please remove this from the PR. It must've been added by mistake. I'll make a PR that adds it to the gitignore.

File: website/docs/reference/integrations/jira-cloud-plugin-installation.md [hunk 1]
```diff
@@ -31,7 +31,11 @@ You will need a Jira admin user.
 
 ## Installation
 
-If you have an Unleash enterprise license you're welcome to reach out to us at sales@getunleash.io for a free plugin license, otherwise you'll need to try the plugin for 30 days free or purchase a license through the marketplace.
+Start by visiting the [Landing page for the Unleash Enterprise For Jira plugin on Atlassian marketplace](https://marketplace.atlassian.com/apps/1231447/unleash-enterprise-for-jira?hosting=cloud&tab=overview).
```
Comment (sighphyre) [inline website/docs/reference/integrations/jira-cloud-plugin-installation.md:34]: Probably needs a mention that this is only for Pro and Enterprise, if you're on OSS this straight up won't work

File: website/docs/reference/integrations/jira-cloud-plugin-installation.md [hunk 2]
```diff
@@ -31,7 +31,11 @@ You will need a Jira admin user.
...
+
+![Atlassian marketplace landing page for the Unleash Enterprise For Jira plugin.](/img/unleash-marketplace-for-unleash-enterprise-jira-plugin.png)
+
+Click the yellow button in the top right corner labeled Get it now, choose which site to install the app and click the blue button Install app located at the bottom right.
```
Comment (thomasheartman) [inline website/docs/reference/integrations/jira-cloud-plugin-installation.md:38]:
```suggestion
1. Click the yellow button in the top right corner labeled "Get it now"
2. Choose which site to install the app to.
3. Click the blue "Install app" button located at the bottom right.
```

**Review by thomasheartman — APPROVED** [review]
This looks good to me once you remove the auto-generated file 😄 Good job!

File: website/docs/reference/integrations/jira-cloud-plugin-installation.md (hunk 1 above)
Comment (thomasheartman) [reply website/docs/reference/integrations/jira-cloud-plugin-installation.md:34]: Ah, this is a very good point!

File: website/docs/reference/integrations/jira-cloud-plugin-installation.md (hunk 2 above)
Comment (sighphyre) [inline website/docs/reference/integrations/jira-cloud-plugin-installation.md:38]: This is a bit tough for visually impaired folks to follow, we're generally pretty good in our docs about that. Not sure if we have a way to handle this here

Comment (sighphyre) [reply website/docs/reference/integrations/jira-cloud-plugin-installation.md:38]: Ha! Thomas has already given you some suggestions, ignore me

File: website/docs/reference/integrations/jira-cloud-plugin-installation.md (hunk 1 above)
Comment (daveleek) [reply website/docs/reference/integrations/jira-cloud-plugin-installation.md:34]: Thank you, yes, very good point!

File: website/docs/reference/integrations/jira-cloud-plugin-installation.md (hunk 2 above)
Comment (thomasheartman) [reply website/docs/reference/integrations/jira-cloud-plugin-installation.md:38]: Yeah, I saw that, but we do provide the labels for the buttons as well. So as far as I know, that should cover it. I don't know what else we could do. Have any other suggestions?

Comment (thomasheartman) [reply website/docs/reference/integrations/jira-cloud-plugin-installation.md:38]: Yeah. The screenshot doesn't provide much extra detail. I think the alt text is fine too. From my (admittedly limited) knowledge, I think we've gotten the best of both worlds here: both described the label for visually impaired folks and described the location and color for those who benefit from that. So kudos to @daveleek for getting them both in 🙌🏼

**Review by thomasheartman — APPROVED** [review]

**Review by sighphyre — APPROVED** [review]
G

**Pull request merged**